vhdbuilder/packer/components.json

```diff
@@ -419,11 +419,10 @@
     },
     {
       "downloadURL": "mcr.microsoft.com/aks/command/runtime:*",
-      "amd64OnlyVersions": [
-        "master.210524.9",
-        "master.211013.1"
```

Comment:
master.211013.1 was not used by RP anymore since Dec 7 2021, master.210524.9 was not used by RP even earlier.